spec/std/math_spec.cr

```diff
@@ -41,6 +42,18 @@ describe "Math" do
       Math.sqrt(4_f32).should eq(2)
       Math.sqrt(4).should eq(2)
     end
+
```

Comment:
Maybe this should better be in the big* spec?

Reply:
@straight-shoota Idk, but spec for json+big and yaml+big are under json and yaml

Reply:
I think this is better in the `big` specs.